DOCS/man/console.rst

```diff
@@ -169,6 +169,11 @@ Configurable Options
 
     Set the font border size used for the REPL and the console.
 
+``margin_y``
```

Comment:
Why only `margin_y`?

Reply:
I didn't know if margin_x is needed, I guess I will add it.